test/kwtool_web/controllers/auth_controller_test.exs

```diff
@@ -0,0 +1,10 @@
+defmodule KwtoolWeb.AuthControllerTest do
+  use KwtoolWeb.ConnCase, async: true
+
+  describe "get the sign up route" do
```

Comment:
The `describe` block must use the method name and rarity:

```suggestion
  describe "sign_up/2" do
```

Reply:
I updated it.

Reply:
Sorry, I forgot to add that it needs to be prefixed with the HTTP verb 🙈 like @hanam1ni commented